src/util.cpp

```diff
@@ -584,14 +581,14 @@ void AllocateFileRange(FILE *file, unsigned int offset, unsigned int length) {
 #ifdef WIN32
 fs::path GetSpecialFolderPath(int nFolder, bool fCreate)
 {
-    char pszPath[MAX_PATH] = "";
+    WCHAR pszPath[MAX_PATH] = L"";
```

Comment:
```suggestion
    WCHAR pszPath[MAX_PATH] = L"";
```
Rename the variable to `pwszPath` :-p